demos/table2.php

```diff
@@ -3,33 +3,70 @@
 date_default_timezone_set('UTC');
 include 'init.php';
 
+$app->add(['Header', 'Table with various headers', 'subHeader'=>'Demonstrates how you can add subheaders, footnotes and other insertions into your data table', 'icon'=>'table']);
```

Comment:
table2 demo page is not included in left side menu.

Reply:
also opening that demo page throws exception
```
Error: Class '\atk4\data\Field\Callback' not found (in C:\wamp64\www\git\atk\ui\vendor\atk4\core\src\FactoryTrait.php:153)
#0 C:\wamp64\www\git\atk\ui\vendor\atk4\data\src\Model.php(373): atk4\data\Model->factory(Array, Array, '\\atk4\\data\\Fiel...')
#1 C:\wamp64\www\git\atk\ui\demos\table2.php(49): atk4\data\Model->addField('amount_copy', Array)
#2 {main}
```
If it doesn't do that on your environment, then try running `composer update`

Reply:
fixed both.